app/templates/base.html

```diff
@@ -2,7 +2,7 @@
 {%- from 'govuk_frontend_jinja/components/error-summary/macro.html' import govukErrorSummary -%}
 {%- from 'govuk_frontend_jinja/components/notification-banner/macro.html' import govukNotificationBanner -%}
 {%- from 'govuk_frontend_jinja/components/phase-banner/macro.html' import govukPhaseBanner -%}
-{% set assetPath = url_for('static', filename='').rstrip('/') %}
+{% set assetPath = url_for('static', filename='assets').rstrip('/') %}
```

Comment:
Becaudse we moved the assets above, this had to change, although we could actually momit this now since the default for assetPath is `"assets"`: https://github.com/LandRegistry/govuk-frontend-jinja/blob/main/govuk_frontend_jinja/templates/template.html